src/stores/toolbarStore.ts

```diff
@@ -1,41 +1,77 @@
 import { observable, computed, action } from 'mobx';
+import _ from 'lodash';
 import ToolbarTool from '~/enums/toolbarTool';
 import EditMode from '~/enums/editMode';
 
+import BaseTool from '~/tools/BaseTool';
+import AddNetworkNodeTool from '~/tools/AddNetworkNodeTool';
+import AddNewRoutePathTool from '~/tools/AddNewRoutePathTool';
+import CopyTool from '~/tools/CopyTool';
+import DivideLinkTool from '~/tools/DivideLinkTool';
+import EditNetworkNodeTool from '~/tools/EditNetworkNodeTool';
+import PrintTool from '~/tools/PrintTool';
+
+const TOOLS = [
+    new AddNetworkNodeTool(),
+    new AddNewRoutePathTool(),
+    new CopyTool(),
+    new DivideLinkTool(),
+    new EditNetworkNodeTool(),
+    new PrintTool(),
+];
+
+// TODO: Rename ToolbarStore -> toolStore?
```

Comment:
Hmm, I dont think toolStore is specific enough.

Reply:
I just see TODO:s as something that needs to be fixed. Renaming it to toolStore is not something that needs to be fixed, it is something that could be discussed. And if we have that discussion, I would probably suggest MapTools as a name.

Reply:
If i add a question mark that usually means "this probably needs to be fixed"

But yes you are right, mapTools could be a better name than just tool. Maybe rename enums/toolbarTool -> enums/mapTool

But as this isn't 100% sure yet we can remove this comment